bedrock/newsletter/urls.py

```diff
@@ -21,8 +21,9 @@
     path("newsletter/updated/", views.updated, name="newsletter.updated"),
     # Confirm subscriptions
     re_path("^newsletter/confirm/(?P<token>%s)/$" % uuid_regex, views.confirm, name="newsletter.confirm"),
```

Comment:
This is the only place that still uses the `uuid_regex` defined above. Do we still need that or can we use the simpler regex you've updated below?

Reply:
I'm not a regex expert, but for `/country/` below I copied what we already use for `/existing/`, which supports making the token optional. Here we don't need to rewrite the `/confirm/` URL to remove the token, since it now redirects instead. So I think we're probably OK to keep this slightly more strict check here?